index.html

```diff
@@ -50,126 +50,424 @@ <h1>Hey There.
+            <div class="project pj2">
+              <div class="projectHeader">
+                <h3>Profesional Art Printing Data</h3>
+              </div>
+              <div class="projectBody">
                 <p>
-                    A daily selection of privately personalized reads; 
-                    no accounts or sign-ups required. 
-                    has been the industry's standard
+                  A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard
                 </p>
-                <ul class="grid-container2">
-                    <li class="item1"><a href="#">html</a></li>
-                    <li class="item1"><a href="#">bootstrap</a></li>
-                    <li class="item1"><a href="#">Ruby</a></li>
+                <ul>
+                  <li class="projectHTML">
+                    <ul>
+                      <li>html</li>
+                    </ul>
+                  </li>
+                  <li class="projectBootstrap">
+                    <ul>
+                      <li>bootstrap</li>
+                    </ul>
+                  </li>
+                  <li class="projectRuby">
+                    <ul>
+                      <li>ruby</li>
+                    </ul>
+                  </li>
                 </ul>
-                <button name="button" type="submit" class="btn-2">See Project</button>
+              </div>
+              <div class="projectButton">
+                <button type="button">See Project</button>
+              </div>
             </div>
     
-            <div class="back-images">
-                <div class="background-image"></div>
-                <h2>Profesional Art Printing Data</h2>
+            <div class="project pj3">
+              <div class="projectHeader">
+                <h3>Profesional Art Printing Data</h3>
+              </div>
+              <div class="projectBody">
                 <p>
-                    A daily selection of privately personalized reads; 
-                    no accounts or sign-ups required. 
-                    has been the industry's standard
+                  A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard
                 </p>
-                <ul class="grid-container2">
-                    <li class="item1"><a href="#">html</a></li>
-                    <li class="item1"><a href="#">bootstrap</a></li>
-                    <li class="item1"><a href="#">Ruby</a></li>
+                <ul>
+                  <li class="projectHTML">
+                    <ul>
+                      <li>html</li>
+                    </ul>
+                  </li>
+                  <li class="projectBootstrap">
+                    <ul>
+                      <li>bootstrap</li>
+                    </ul>
+                  </li>
+                  <li class="projectRuby">
```

Comment:
- [ ] Please make sure that all the projects are appearing on the screen and make sure they looks exactly like Figma design 
![image](https://user-images.githubusercontent.com/74744787/160024544-31e1ec50-ecf8-4a8a-8a0a-22e7a65417b7.png)

![image](https://user-images.githubusercontent.com/74744787/160024575-dbc25529-28fe-41ab-ad1a-f26ded68d5f8.png)

I think the problem here comes from the quires make sure that you are using them correctly 
![image](https://user-images.githubusercontent.com/74744787/160024698-3304619b-11f8-42bb-8e13-d078010f07bf.png)